### PR TITLE
Add Modelica language

### PR DESCRIPTION
**Pull request by dietmarw**

### CODE REVIEW

**Comment by dietmarw** [comment]
Any reason why the CI jobs got cancelled?

File: tests/data/Modelica.mo [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// 21 lines 13 code 4 comments 3 blanks
```
Comment (beutlich) [inline tests/data/Modelica.mo:1]:
```suggestion
// 21 lines 13 code 5 comments 3 blanks
```

Guess, the header line counts as comment?

Comment (dietmarw) [reply tests/data/Modelica.mo:1]: I'm unsure. I looked at other examples and some do and some don't.

Comment (beutlich) [reply tests/data/Modelica.mo:1]: Just guessed this based on the failure: 4 != 5.

Comment (dietmarw) [reply tests/data/Modelica.mo:1]: Well I guess it is worth a try.

Comment (dietmarw) [reply tests/data/Modelica.mo]: Looks like this was it. Thanks @beutlich

**Pull request merged**